internal/storage/fakestorage_util.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	"github.com/fsouza/fake-gcs-server/fakestorage"
+)
+
+const port uint16 = 8081
+const host string = "127.0.0.1"
+
+const DefaultBucketName string = "gcsfuse-default-bucket"
```

Comment:
nit: lets keep the names simple. please remove default in all variable names and method names.